lib/pka.c

```diff
@@ -2168,3 +2168,41 @@ int pka_dsa_signature_verify(pka_handle_t     handle,
         return pka_submit_cmd(handle, user_data, CC_DSA_VERIFY, &operands);
 }
 
+int pka_get_rand_bytes(pka_handle_t  handle,

```

Comment:
To be consistent wit the rest of the APIs, we might have to pass user_data as well. This might encapsulate private data or info about the caller or request issuer.